workspaces/sonarqube/plugins/sonarqube/README.md

```diff
@@ -10,14 +10,15 @@ The SonarQube Plugin displays code statistics from [SonarCloud](https://sonarclo
 
 ```bash
 # From your Backstage root directory
-yarn --cwd packages/app add @backstage-community/plugin-sonarqube
+yarn --cwd packages/app add @backstage-community/plugin-sonarqube @backstage-community/plugin-sonarqube-react
 ```
 
 2. Add the `EntitySonarQubeCard` to the EntityPage:
 
 ```diff
   // packages/app/src/components/catalog/EntityPage.tsx
 + import { EntitySonarQubeCard } from '@backstage-community/plugin-sonarqube';
++ import { isSonarQubeAvailable } from '@backstage-community/plugin-sonarqube-react';
```

Comment:
It's really too bad this is deprecated in the frontend plugin and you need to add the `-react` plugin just for this one part for a essentially standard install and setup. Not your fault, this change makes sense to have 👍